vck/src/commonMain/kotlin/at/asitplus/wallet/lib/data/rfc/tokenStatusList/StatusListView.kt

```diff
@@ -0,0 +1,108 @@
+package at.asitplus.wallet.lib.data.rfc.tokenStatusList
+
+import at.asitplus.wallet.lib.data.rfc.tokenStatusList.primitives.TokenStatus
+import at.asitplus.wallet.lib.data.rfc.tokenStatusList.primitives.TokenStatusBitSize
+import kotlin.experimental.and
+import kotlin.experimental.or
+
+/**
+ * Utility class to access token statuses by index from an uncompressed byte array.
+ * In theory this should improve storage size, since byte array allows for a compressed
```

Comment:
seems like there is at least one word missing at the end?

Reply:
fixed in 93ce860696aa4077acb9a053a8b3eac9bee2733b